plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import Typography from '@mui/material/Typography';
 import { JiraDataResponse } from '@axis-backstage/plugin-jira-dashboard-common';
-import { ErrorPanel, Table } from '@backstage/core-components';
+import {
+  ErrorPanel,
+  InfoCard,
+  Table,
+  TableFilter,
+} from '@backstage/core-components';
 import { capitalize } from 'lodash';
 import { columns } from './columns';
 
 type Props = {
   tableContent: JiraDataResponse;
+  filters?: TableFilter[];
 };
 
-export const JiraTable = ({ tableContent }: Props) => {
+export const JiraTable = ({ tableContent, filters }: Props) => {
```

Comment:
Nice! Like that the filters are customizable.

Reply:
Sounds good, I can update it :) My only question would be what fields should be included in the default filters? Currently Priority and Status are enabled but is assignee also needed?

Reply:
Does it add any complexity to add the assignees? Otherwise I would say that I think that it could be useful :)

Reply:
I investigated this and it would require amending the Jira response on the backend.
Unassigned tickets assignee fields are null which results in an error when they are parsed for use by the table filters.
The field could be amended so that the default value is "unassigned" rather than null however this would change the current behaviour. I suggest this is raised as a separate issue if the functionality is wanted

Reply:
Lets do so! Good call! :+1: